src/components/map/map.jsx

```
@@ -0,0 +1,57 @@
+import "maplibre-gl/dist/maplibre-gl.css";
+import MapPin from "./map-pin";
+import {
+  Map as MapLibre,
+  Marker,
+  NavigationControl,
+} from "react-map-gl/maplibre";
+import { centreOfFoxleyWood } from "../../helpers/parse-meeting-point";
+import * as style from "./map.module.css";
+import process from "process";
+
+const Map = ({
```

Biome (2.4.6)

[error] Do not shadow the global "Map" property.

(lint/suspicious/noShadowRestrictedNames)